.github/workflows/ccpp.yml

```diff
@@ -1,29 +1,29 @@
 name: C/C++ CI
 
-on: [push]
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
 
 jobs:
   build:
 
     runs-on: ubuntu-latest
-    
+
     steps:
-    - uses: actions/checkout@v1
-    - name: build and install ELL
-      run: |
-        git clone git://git.kernel.org/pub/scm/libs/ell/ell.git
-        cd ell
-        ./bootstrap
-        ./configure --prefix=`dirname $PWD`
-        make install
-      working-directory: ./ell-install
+    - uses: actions/checkout@v2
+    - name: dependencies
+      run: sudo apt install autoconf-archive libell-dev pandoc
```

Comment:
@ossama-othman no need to use `-y` here?

Reply:
I don't believe so.  I've been able to install packages without `-y` in the GitHub Action in another of my GitHub repositories.

Reply:
On a related note, it would probably be better to switch to `apt-get` since it has a stable CLI.  I get the following warning when using `apt`:
```
WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
```

Reply:
I created a pull request that uses `apt-get -y` instead of `apt`.  See #131

Reply:
ah yes, I didn't think about that, `apt` is maybe OK without `-y`. I never use it in my script. That's nice they added a warning :)